platformpolicy/crypto.go

```diff
@@ -14,15 +14,13 @@
 // limitations under the License.
 //
 
-package sign
+package platformpolicy
 
-import (
-	"crypto"
+const (
+	COMMON_CRYPTO cryptoStruct = "COMMON_CRYPTO"
```

Comment:
Let's use `CamelCase` in variable names.

Reply:
And please use iota here. Strings are not needed here

Reply:
done